test/functionalTest/cases/3189_oneshot_subscription/3189_oneshot_subscription.test

```diff
@@ -0,0 +1,313 @@
+# Copyright 2018 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Oneshot subscription with POST /v2/entities, POST /v2/subscriptions, PATCH /v2/entities/id/attrs, GET /v2/subscriptions/subscription_id, PATCH /v2/subscriptions/subscription_id and PATCH /v2/entities/id/attrs
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print
+
+--SHELL--
+
+#
+# 01. POST /v2/entities, to create Room1 with temperature and pressure.
+# 02. POST /v2/subscriptions, to create subscription with status oneshot.
+# 03. PATCH /v2/entities/Room1/attrs, to update the value of pressure attribute.
```

Comment:
I'll suggest to add an additional entity update after this one to ensure that just one notification is got at dump ste (a regular subscription will involve two notifications).

Reply:
fixed in 1211a29